isis/src/base/objs/SpiceRotation/SpiceRotation.cpp

```diff
@@ -1222,15 +1303,32 @@ namespace Isis {
    * Set the rotation angles (phi, delta, and w) for the current time to define the
    * time-based matrix CJ. This method was created for unitTests and should not
    * be used otherwise.  It only works for cached data with a cache size of 1.
-   *
+   *  
+   * // FIXME - MAYBE THIS SHOULD BE IN THE UNITTEST THEN.... (see comment) 
```

Comment:
Why is this here?

Reply:
Thanks for catching this. This is a note I made to myself that I didn't end up changing. There's a comment in this function that it was only added for unit testing and shouldn't be used for anything else... so I thought it might make sense to just subclass SpiceRotation, add this function, and use that for the unit test. Then I realized that this is going to be gtest converted at some point anyway, and going to the extra effort of moving this probably wasn't worth it.

Reply:
So, I'm just going to delete the comment.